fontbakery-check-ttf.py

```diff
@@ -2044,6 +2044,27 @@ def ff_check(condition, description, err_msg, ok_msg):
             fb.error(error_message.format(font_file))
 
     # ----------------------------------------------------
+    fb.new_check("Font contains the first few mandatory glyphs"
+                 " (.null, CR, space)?")
+    # It would be good to also check
+    # for .notdef (codepoint = unspecified)
+    null = getGlyph(font, 0x0000)
+    CR = getGlyph(font, 0x000D)
+    space = getGlyph(font, 0x0020)
+
+    missing = []
+    if null is None: missing.append(".null (0x0000)")
+    if CR is None: missing.append("CR (0x00D)")
+    if space is None: missing.append("space (0x0020)")
+    if missing != []:
+        fb.error(("Font is missing"
+                  " the following mandatory glyphs:"
+                  " {}.").format(", ".join(missing)))
+    else:
+        fb.ok("Font contains the first few mandatory glyphs"
+              " (.null, CR, space).")
```

Comment:
I don’t think this is what @davelab6 is asking for, you need to check that the 4 glyphs are there **and** that they are the first 4 glyphs (i.e. glyph ids 0 to 3) in the font, and you probably need to check that they have the expected name as well.

Though personally I think that requirement in the spec is bogus and nothing really requires it. Checking that there is a `.notdef` glyph with id 0 and no Unicode value makes sense, though.